README.md

```diff
@@ -4,11 +4,11 @@
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/facebook/hermes/blob/master/CONTRIBUTING.md)
 <img src="./website/static/img/logo.svg" alt="Hermes logo - large H with wings" align="right" width="20%"/>
 
-Hermes is a JavaScript engine optimized for fast start up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
+Hermes is a JavaScript engine optimized for fast start-up of [React Native](https://facebook.github.io/react-native/) apps on Android. It features ahead-of-time static optimization and compact bytecode.
 
 If you're only interested in using pre-built Hermes in a new or existing React Native app, you do not need to follow this guide or have direct access to the Hermes source. Instead, just follow [these instructions to enable Hermes](https://facebook.github.io/react-native/docs/hermes).
 
-If you want to know how to build and hack on Hermes directly, and/or integrate Hermes built from source into a React Native app then read on.
+If you want to know how to build and hack on Hermes directly, and/or integrate Hermes built from the source into a React Native app then read on.
```

Comment:
I don't think this change is necessary - "source" is frequently used without the definite article, particularly in this context.